static/js/components/tiles/disaster_event_map_tile.tsx

```diff
@@ -70,6 +71,8 @@ const CSS_SELECTOR_PREFIX = "disaster-event-map";
 const REDIRECT_URL_PREFIX = "/disasters/";
 const MAP_POINTS_MIN_RADIUS = 1.5;
 const MAP_POINTS_MIN_RADIUS_EARTH = 0.8;
+// Set of dcids of places that should use the selected place as the base geojson
+const PLACE_MAP_PLACES = new Set(["country/AUS"]);
```

Comment:
do we need this map if we have the fallback in place?

Reply:
yes, we need this because we actually do get a few places when getting children for country/AUS ..